model/jpa/src/main/resources/META-INF/jpa-changelog-master.xml

```diff
@@ -73,5 +73,5 @@
     <include file="META-INF/jpa-changelog-15.0.0.xml"/>
     <include file="META-INF/jpa-changelog-17.0.0.xml"/>
     <include file="META-INF/jpa-changelog-18.0.0.xml"/>
-
+    <include file="META-INF/jpa-changelog-19.0.0.xml" />
```

Comment:
In order to avoid changing the database schema, could you use realm attributes rather than adding a column?